modules/system/classes/VersionManager.php

```diff
@@ -544,6 +548,54 @@ protected function extractScriptsAndComments($details)
             $scripts = [];
         }
 
-        return array($comments, $scripts);
+        return [$comments, $scripts];
+    }
+
+    /**
+     * Get the current version plugin
+     *
+     * @param $plugin
+     * @return int
```

Comment:
I believe this returns a string.
```suggestion
     * @return string
```